src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java

```diff
@@ -93,46 +103,65 @@ public ConcurStandardAccountingExtractCollectorBatchBuilder(
         this.notificationEmail = parameterService.apply(ConcurParameterConstants.CONCUR_SAE_COLLECTOR_NOTIFICATION_CONTACT_EMAIL);
         this.notificationPerson = parameterService.apply(ConcurParameterConstants.CONCUR_SAE_COLLECTOR_NOTIFICATION_CONTACT_PERSON);
         this.notificationPhone = parameterService.apply(ConcurParameterConstants.CONCUR_SAE_COLLECTOR_NOTIFICATION_CONTACT_PHONE);
+        this.prepaidOffsetAccountNumber = parameterService.apply(ConcurParameterConstants.CONCUR_SAE_COLLECTOR_PREPAID_OFFSET_ACCOUNT_NUMBER);
+        this.prepaidOffsetObjectCode = parameterService.apply(ConcurParameterConstants.CONCUR_SAE_COLLECTOR_PREPAID_OFFSET_OBJECT_CODE);
+        this.cashOffsetObjectCode = parameterService.apply(ConcurParameterConstants.CONCUR_SAE_COLLECTOR_CASH_OFFSET_OBJECT_CODE);
+        
+        reset();
     }
 
-    public void reset() {
+    protected void reset() {
         this.collectorBatch = new CollectorBatch();
-        this.originEntries = new LinkedHashMap<>();
+        this.lineGroups = new LinkedHashMap<>();
         this.nextFakeObjectCode = 1;
         this.nextTransactionSequenceNumbers = new HashMap<>();
+        this.batchSequenceNumber = 0;
+        this.reportData = null;
     }
 
     public CollectorBatch buildCollectorBatchFromStandardAccountingExtract(int nextBatchSequenceNumber,
-            ConcurStandardAccountingExtractFile saeFileContents) {
+            ConcurStandardAccountingExtractFile saeFileContents, ConcurStandardAccountingExtractBatchReportData newReportData) {

```

Comment:
Given that IllegalArgumentException is an unchecked exception, I don't think it's strictly necessary to include it in the "throws" clause. I'll at least update the Javadocs, though.